guides/source/asset_pipeline.md

```diff
@@ -3,81 +3,60 @@
 The Asset Pipeline
 ==================
 
-This guide covers the asset pipeline.
+This guide covers the asset pipeline as it was implemented by Sprockets.
```

Comment:
Doesn't this describe the Import Maps and Sprockets guide?
```suggestion
This guide covers the asset pipeline using Import Maps and Sprockets.
```

Maybe also link to the Sprockets guide:
```
The Sprockets only asset pipeline guide has been moved to a seperate guide.
```